vcd/resource_vcd_vapp_network.go

```diff
@@ -92,18 +91,6 @@ func resourceVcdVappNetwork() *schema.Resource {
 				Optional:    true,
 				Description: "org network name to which vapp network is connected",
 			},
-			"firewall_enabled": {
```

Comment:
please remind me, why these are removed

Reply:
These properties were marked as "Removed" and should have been deleted in 3.0.
Now the `Removed` field has been itself removed, and we have no choice but to eliminate the property

Reply:
ahf. ok - Great ;-)